### DISCUSSION TITLE
setting date, correspondent, title, tag by naming files and feeding it to the consumer folder?

### DISCUSSION THREAD

**Discussion by cgovella** [jonaswinkler/paperless-ng#953] [Support]
Hi I've been using the old paperless archived until now- I'm experimenting with the paperless-ng, which is sharp looking. I am in the habit of naming all my scanned PDF files in the 'Date - Correspondent - Title - Tags.pdf' pattern, which in the old version, would set my metadata without having to use the GUI. Does this behavior exist in paperless-ng? I've been looking at the settings page in the new GUI and I don't see anything there. Thanks,

**Comment by who-knows-it** [comment]
Hi, it would also be helpfull if tags added to pdfs before import could be retrieved from the meta data when importing to paperless.

**Comment by cgovella** [comment]
haven't heard anything yet, any info on how importing metadata via reading the filenames works in paperless-ng?
